src/lib/client-file-uploader.js

```diff
@@ -173,6 +177,17 @@ export async function uploadImportSqlFileToS3( {
 		debug( `** Compression resulted in a ${ calculation } smaller file 📦 **\n` );
 	}
 
+	const maxFileSize = env.launched
```

Comment:
I'm ok if we choose to check this in the gates rather than here, but we should be aware it's not the same check we're performing server-side.